release notes/v0.32.0.md

```diff
@@ -0,0 +1,108 @@
+- updating the gRPC libraries fixed bug ([#1928](https://github.com/k6io/k6/issues/1928)) and probably others. ([#1937](https://github.com/k6io/k6/pull/1937))
+
+### `ArrayBuffer` is now supported in all JS APIs dealing with binary data, including in WebSocket messages ([#1841](https://github.com/k6io/k6/pull/1841))
+
+Besides the minor breaking changes (see the "Breaking changes" section below), it's now possible to send binary WS messages with the `socket.sendBinary()` function and to receive binary messages with the `binaryMessage` event handler:
+
+  ```javascript
+  const binFile = open('./file.pdf', 'b');
+
+  export default function () {
+    ws.connect('http://wshost/', function(socket) {
+      socket.on('open', function() {
+        socket.sendBinary(binFile);
+      });
+
+      socket.on('binaryMessage', function(msg) {
+        // msg is an ArrayBuffer, so we can wrap it in a typed array directly.
+        new Uint8Array(msg);
+      });
+    });
+  }
+  ```
+
+### Official arm64 releases for macOS and Linux ([#2000](https://github.com/k6io/k6/pull/2000))
+
+We will now publish binary arm64 releases for macOS and Linux for new k6 releases. Support for these new architectures should be stable, given Go's cross-platform compatibility, but please report any issues you experience.
+
+## Other enhancements and UX improvements
+
+- Options: k6 will now warn you on unrecognised configuration JS options in most cases instead of just silently ignoring them. ([#1919](https://github.com/k6io/k6/pull/1919))
+- error_code: the tag `error_code` should now be set more accurately in some cases. ([#1952](https://github.com/k6io/k6/pull/1951))
+
+## Bugs fixed!
+
+- Arrival-rate executors could in some cases report twice as many used VUs than what was actually true. ([#1954](https://github.com/k6io/k6/issues/1954)) fixed by ([#1955](https://github.com/k6io/k6/pull/1955))
+- In cases of an error while reading the response body, the newly added `responseCallback` in v0.31.0 would be evaluated with the returned status code, while the reported one would be `0`, as the response errored out and k6 does not return incomplete responses. Now `responseCallback` will also receive a `0` status. ([#1962](https://github.com/k6io/k6/pull/1962))
+- Fix Kafka output not being usable with the InfluxDB format after v0.31.0 changes. ([#1914](https://github.com/k6io/k6/pull/1914))
+- Error out with a user friendly message if `ramping-vus` executor would've not run a single iteration instead of just doing nothing. ([#1942](https://github.com/k6io/k6/pull/1942))
+
+## Internals
+
+- JS: Added a way for JS modules to have per VU initialization and object. This can also be used by xk6 modules by implementing `github.com/loadimpact/k6/js/modules#HasModuleInstancePerVU`. (Part of ([#1911](https://github.com/k6io/k6/pull/1911)))
```

Comment:
```suggestion
- JS: Added a way for JS modules to have per VU initialization and object. This can also be used by xk6 modules by implementing `github.com/loadimpact/k6/js/modules#HasModuleInstancePerVU`. (Part of [#1911](https://github.com/k6io/k6/pull/1911))
```

wdyt @imiric